### PR TITLE
Updating the readme

### PR DESCRIPTION
**Pull request by iamanissa**

### CODE REVIEW

**Comment by osondoar** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -1,24 +1,46 @@
...
+***
+## Getting Started
+Fork this repo
+Clone your repo
+```bash
+git clone yourrepo
```
Comment (osondoar) [inline README.md]: what about using the actual repo's url here?

Comment (iamanissa) [reply README.md]: I don't know the other persons repo url.

Comment (jmkoni) [reply README.md]: I think what is mean is to put `git clone git@github.com/youusername/justice.git`

**Comment by jmkoni** [comment]
This fixes issue #10

File: README.md (hunk 1 above)
Comment (iamanissa) [reply README.md]: oh I see 👍

**Pull request merged**